cmd/lakefs/cmd/run.go

```diff
@@ -212,6 +219,24 @@ func printWelcome(w io.Writer) {
 	_, _ = fmt.Fprintf(w, "Version %s\n\n", version.Version)
 }
 
+var localWarningBanner = `
+ ██       ██                           ██              	  
+░██      ░██                          ░░                  ░██
+░██   █  ░██  ██████   ██████ ███████  ██ ███████   █████ ░██
+░██  ███ ░██ ░░░░░░██ ░░██░░█░░██░░░██░██░░██░░░██ ██░░░██░██
+░██ ██░██░██  ███████  ░██ ░  ░██  ░██░██ ░██  ░██░██  ░██░██
+░████ ░░████ ██░░░░██  ░██    ░██  ░██░██ ░██  ░██░░██████░░ 
+░██░   ░░░██░░████████░███    ███  ░██░██ ███  ░██ ░░░░░██ ██
+░░       ░░  ░░░░░░░░ ░░░    ░░░   ░░ ░░ ░░░   ░░   █████ ░░ 
```

Comment:
A bit much, given our quickstart will also show this. I'd do without the big banner..